src/main/java/com/minecolonies/coremod/colony/workorders/AbstractWorkOrder.java

```diff
@@ -132,6 +134,7 @@ public static AbstractWorkOrder createFromNBT(@NotNull final NBTTagCompound comp
     public void readFromNBT(@NotNull final NBTTagCompound compound)
     {
         id = compound.getInteger(TAG_ID);
+        priority = compound.getInteger(TAG_TH_PRIORITY);
         claimedBy = compound.getInteger(TAG_CLAIMED_BY);
     }
```

Comment:
still needs a compound.hasKey and the formatting of order.setPriority needs the same and formatting

Reply:
Yep.

Reply:
I'm not following!   Please explain.

Reply:
If we load a save which does not have the priority tag set, it will crash.
So you have to check if the tag exists first.
With: compound.hasKey